kyaml/fn/runtime/runtimeutil/functiontypes.go

```diff
@@ -33,6 +34,78 @@ const (
 	UserNobody ContainerUser = "nobody"
 )
 
+const defaultEnvValue string = "true"
+
+// ContainerEnvs contains the environment variables for the container
+type ContainerEnvs struct {
+	// EnvsMap is a key-value map that will be set as env in container
+	EnvsMap map[string]string `json:"envsMap,omitempty" yaml:"envsMap,omitempty"`
+
+	// ExportKeys are only env key. Value will be the value in the host system
```

Comment:
Are these a subset of the keys in EnvVars?

Reply:
No, EnvsVar contains key-value map. ExportedKeys contains the env key that user wants to expose to container from host system. No value specified.

Reply:
So maybe call this `VarsToExport`.